### ISSUE TITLE
CH's "hide_tabs" used with HA's "visible: false" causes page's to scroll to top

### ISSUE THREAD

**Issue by Giel88** [maykar/custom-header#256]
When using hide_tabs and when you scroll to the bottom of the page the page jumps back to top of the page after x seconds. When **not** using hide_tabs this doesn't seem to happen.

**Comment by Giel88** [comment]
- No (relevant) errors in HA log
- No (relevant) errors in browser console
- HassOS 4.12 / Home Assistant 0.114.3 / Custom Header v1.7.2
- Affected browsers: Chrome Mobile, Chrome Desktop, Android HA official app.
- Installed using HACS

```
compact_mode: true
background: rgba(0, 0, 0, 0)
elements_color: var(--app-header-text-color)
hide_tabs: '0'
```

**Comment by maykar** [comment]
Using your config I can't replicate this issue. Scrolled to the bottom of the page on multiple devices (iOS, Android, HA apps, Chrome, Firefox, and Safari) on multiple different views and waited 10 minutes. All stayed at the bottom of the page.

Does this only happen on certain views? Do those tabs have something in common?

**Comment by Giel88** [comment]
Tried some different things; only views with custom cards, views with standard cards, adding a second view to the dashboard, etc.

And as soon as I set the second view of the dashboard to 'visible: false' this behavious happens.

**Comment by maykar** [comment]
Cool, I can reproduce it. So it only seems to happen on the tab that is set to `visible: false`, correct?

Seems to affect desktop browsers too. Will look into it more and see what I can do.

**Comment by Giel88** [comment]
> Cool, I can reproduce it. So it only seems to happen on the tab that is set to `visible: false`, correct?

No, it happens on every view in the dashboard. So if one of the views contains `visible: false` all views will automatically scroll to top. Not just the one that's set to `visible: false`

**Comment by maykar** [comment]
Hmmm.. Seems to only happen to me on the tab that has it set, but it's buggy behavior regardless. Does adding `hidden_tab_redirect: false` to your CH config fix this issue?

Also, out of curiosity, why use 'visible: false` over hiding the tab in CH with `hide_tabs`?

**Comment by maykar** [comment]
Nevermind, believe I've found the culprit. hidden_tab_redirect won't solve it, but I'm working on a fix now.

Thank you!

**Comment by Giel88** [comment]
You're welcome!

> Also, out of curiosity, why use visible: false over hiding the tab in CH with hide_tabs?

I was looking for a way to use swipe navigation, exclude some of these views from the swipe navigation and also hide all tabs. I know you can achieve this by hiding tabs, and skipping them in the swipe navigation but that requires changes in two different spot instead of just setting a view to: `visible: false` :)

**Comment by maykar** [comment]
Should be fixed in latest release 1.7.3
